cfme/cloud/instance/openstack.py

```diff
@@ -54,6 +54,44 @@ def ui_powerstates_unavailable(self):
             'on': [self.START],
             'off': [self.SUSPEND, self.SOFT_REBOOT, self.HARD_REBOOT]}
 
+    def attach_volume(self, name, mountpoint=None):
+        initial_volume_count = self.volume_count
+        view = navigate_to(self, 'AttachVolume')
+        view.form.volume.select_by_visible_text(name)
+        if mountpoint:
```

Comment:
Required: this is not necessary, as if mountpoint=None if will not be filled by default

Reply:
Also tried without if condition and None as default and error message that None is invalid